### PR TITLE
People changes additional tweaks

### PR DESCRIPTION
**Pull request by johniak**

### CODE REVIEW

**Review by patryk-dabrowski — DISMISSED** [review]

**Review by patryk-dabrowski — APPROVED** [review]

File: backend/hct_mis_api/apps/household/celery_tasks.py [hunk 1]
```diff
@@ -64,26 +67,31 @@ def recalculate_population_fields_chunk_task(households_ids: List[UUID]) -> None
...
     if household_ids:
         params["pk__in"] = household_ids
+    recalculate_composition = None
+    if program_id:
+        program = Program.objects.get(id=program_id)
+        recalculate_composition = program.data_collecting_type.recalculate_composition
+    queryset = Household.objects.filter(**params).only("pk").order_by("pk")
+    if recalculate_composition is None:
+        queryset = queryset.filter(collect_individual_data__in=(COLLECT_TYPE_FULL, COLLECT_TYPE_PARTIAL))
+    elif not recalculate_composition:
```
Comment (MarekBiczysko) [inline backend/hct_mis_api/apps/household/celery_tasks.py:85]: this covers what? recalculate_composition === blank "" ?

Comment (johniak) [reply backend/hct_mis_api/apps/household/celery_tasks.py:85]: False

File: backend/hct_mis_api/apps/household/celery_tasks.py [hunk 2]
```diff
@@ -64,26 +67,31 @@ def recalculate_population_fields_chunk_task(households_ids: List[UUID]) -> None
...
-        .order_by("pk")
-    )
     if queryset.exists():
         paginator = Paginator(queryset, config.RECALCULATE_POPULATION_FIELDS_CHUNK)
 
         for page_number in paginator.page_range:
             page = paginator.page(page_number)
             recalculate_population_fields_chunk_task.delay(
-                households_ids=list(page.object_list.values_list("pk", flat=True))
+                households_ids=list(page.object_list.values_list("pk", flat=True)), program_id=program_id
```
Comment (MarekBiczysko) [inline backend/hct_mis_api/apps/household/celery_tasks.py:94]: program_id can be None at this point, this is correct?

**Review by MarekBiczysko — APPROVED** [review]

**Review by pavlo-mk — APPROVED** [review]

File: backend/hct_mis_api/apps/household/celery_tasks.py (hunk 2 above)
Comment (johniak) [reply backend/hct_mis_api/apps/household/celery_tasks.py:94]: it is ok default value, is also None

File: backend/hct_mis_api/apps/household/celery_tasks.py (hunk 1 above)
Comment (MarekBiczysko) [reply backend/hct_mis_api/apps/household/celery_tasks.py:85]: aaa ok 👍

**Pull request merged**